src/index.html

```diff
@@ -5,9 +5,47 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img class="card__img"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
```

Comment:
apply code styling recommendations from a checklist. The same issue is for `<a>` tag
```suggestion
      <img 
        class="card__img"
        src="images/imac.jpeg"
        alt="imac"
      >
```